src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -234,6 +235,13 @@ class DiskImageEditor extends Component {
     }
   }
 
+  changeBootable (event) {
+    const target = event.target
+    const value = target.type === 'checkbox' ? target.checked : target.value
+    this.setState((state) => ({ values: { ...state.values, bootable: value } }))
```

Comment:
Nit-pick: if rename `value` to `bootable`, then line will look like that `this.setState((state) => ({ values: { ...state.values, bootable } }))`